base/student.rb

```diff
@@ -1,10 +1,10 @@
 require_relative 'person'
 
 class Student < Person
-  attr_reader :classroom
+  attr_accessor :name, :classroom
```

Comment:
- [ ] you don't want to make it possible to change the classroom instance variable outside of this class, therefore I advise you to make use of the attr_reader instead of attr_accessor.

Reply:
well resolved! ✔️